### DISCUSSION TITLE
Vercel Design

### DISCUSSION THREAD

**Discussion by nahtnam** [vercel/community#238] [Show & Tell]
Will the source code for: https://vercel.com/design ever be released?

**Comment by styfle** [comment]
Yes, that is the goal! But there is no ETA at this time.

**Reply by ParagMeshram** [reply]
@styfle any update on this? it would be a really good idea

**Reply by tsathis** [reply]
Is this [geist-ui.dev/](https://geist-ui.dev/) an official one?

https://github.com/geist-org/style

**Reply by styfle** [reply]
@tharindusathis No, that is not official

**Reply by timsully** [reply]
@styfle Any update on this? Love the UI!

**Reply by Shawnxkwang** [reply]
any update? love Vercel's design system

**Reply by AdaptCharm** [reply]
@styfle Any update?

**Reply by shubham-pixelnx** [reply]
@styfle Any update? Loving the vercel's design system, started using it. 
Wondering if the source code for geist implementation is in progress to be released.

**Reply by CFFQuocHuy-zz** [reply]
honestly, these components are so good

**Reply by quyctd** [reply]
Just curious, does anyone know how geist-ui.dev related to Vercel design, it looks very similar.

**Reply by shubham-pixelnx** [reply]
> Just curious, does anyone know how geist-ui.dev related to Vercel design, it looks very similar.

Probable reason:
- https://vercel.com/blog/zeit-is-now-vercel

**Reply by quyctd** [reply]
@shubham-pixelnx I don't think so, the article does not mention anything related to the design 🤔

**Reply by Shawnxkwang** [reply]
@quyctd geitst-ui.dev is built by some people who love Vercel's design language. they are not directly related at all.

**Reply by leerob** [reply]
Correct, that project is not associated to Vercel at all 👍

**Comment by matthewlilley** [comment]
@styfle Any idea of timeframe? I'd love to see this.

**Comment by uncvrd** [comment]
Jumping in to say that I would love to see this as well :)

**Comment by Koushith** [comment]
we are using https://github.com/geist-org/react in our MVP. we would love to use the official one. when it will be out?

**Comment by Bosphoramus** [comment]
Would love to use it *^*

**Comment by yashrajmishra** [comment]
# PLEASE PROVIDE US

**Comment by quyctd** [comment]
Love the design, hope it will release soon
